tern/analyze/docker/dockerfile.py

```diff
@@ -118,6 +125,27 @@ def parse_from_image(dfobj):
     return image_list
 
 
+def expand_from_images(dfobj):
+    '''Replace all parent_image values with their full tag@sha:digest value.
+    Update the structure dictionary with the same information.
+    dfobj: the Dockerfile object created using get_dockerfile_obj'''
+    # update parent_images
+    images = parse_from_image(dfobj)
+    for i, img in enumerate(images):
+        if general.check_image_string(img['name'] + tag_separator +
+                                      img['tag']):
+            dfobj.parent_images[i] = container.get_image_digest(
+                container.get_image(img['name'] + tag_separator + img['tag']))
```

Comment:
It looks here that you are overwriting the `dfobj.parent_images` with the image in digest form. Nice!